extensions/amp-analytics/0.1/visibility-model.js

```diff
@@ -288,6 +297,9 @@ export class VisibilityModel {
       'loadTimeVisibility': this.loadTimeVisibility_ * 100 || 0,
       'minVisiblePercentage': this.minVisiblePercentage_ * 100,
       'maxVisiblePercentage': this.maxVisiblePercentage_ * 100,
+
+      // Scroll depth (max scroll depth will be added by VisibilityManager)
+      'initialScrollDepth': this.initialScrollDepth_,
```

Comment:
This add the `initalScrollDepth` value to the state, I believe we don't want to expose the exact value.

Reply:
Fixed.